js/repl/src/Svg.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import { css } from 'glamor';
+import React from 'react';
+
+type Props = {
+  className: string,
+  path: string
```

Comment:
Maybe call this component `SvgPath` since it's specifically for rendering a path?